Docs/Get_Started_Tutorial.rst

```diff
@@ -135,20 +138,20 @@ Repeat this operation for other files:
 Step 5: Run Calculation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To run Calculation you have to drag the block *Noise_level_from_traffic* into WPS Builder window.
+To run Calculation you have to drag the block ``Noise_level_from_traffic`` into WPS Builder window.
 
 Then, select the orange blocks and indicate the name of the corresponding table your database, for example :
 
-- Building table name : :literal:`BUILDINGS`
-- Sources table name : :literal:`ROADS2`
-- Receivers table name : :literal:`RECEIVERS`
+- Building table name : ``BUILDINGS``
```

Comment:
Here you should update the layers, there are now well centered, the final map of the first tutorial is a little ugly :)